test/integrations/destinations/google_adwords_enhanced_conversions/dataDelivery/business.ts

```diff
@@ -136,11 +137,14 @@ export const testScenariosForV0API = [
         body: generateProxyV0Payload({
           ...commonRequestParameters,
           params: {
+            developerToken: 'ijkl91011',
+            accessToken: 'google_adwords_enhanced_conversions1',
+            loginCustomerId: '0987654321',
             event: 'Product Added',
             customerId: '1234567888',
             destination: 'google_adwords_enhanced_conversions',
           },
-          endpoint: `https://googleads.googleapis.com/${API_VERSION}/customers/1234567888:uploadConversionAdjustments`,
```

Comment:
why will the endpoint be empty ? we are just moving some code to integ-lib - shouldn't component tests be intact ?

Reply:
The endpoints are defined on the SDK itself. So we don't need the endpoint anymore.

Reply:
Cant we remove endpoint all together

Reply:
I have already removed endpoints.

Reply:
understood, we are using SDK in proxy flow. I overlooked that it's happening in transformation flow - so was expecting the payload that gets sent to server should definitely have an endpoint